apps/voting/test/voting.js

```diff
@@ -83,27 +83,27 @@ contract('Voting App', accounts => {
         it('deciding voting is automatically executed', async () => {
             const action = { to: executionTarget.address, calldata: executionTarget.contract.execute.getData() }
             const script = encodeCallScript([action])
-            const voteId = createdVoteId(await app.newVote(script, '', { from: holder50 }))
+            const voteId = createdVoteId(await app.forward(script, { from: holder50 }))
```

Comment:
It's kind of weird to be replacing `newVote()` with the forwarding functionality; the two should be tested separately (I notice we have just one—"forwarding creates vote"—that should also test the forwarding automatically casts a vote).

Also see comment above on adding a param to `newVote()` instead :).